wpseo-woocommerce.php

```diff
@@ -41,21 +43,21 @@
  *
  * @return void
  */
-function initialize_yoast_woocommerce_seo() {
+function initialize() {
 
 	load_plugin_textdomain( 'yoast-woo-seo', false, dirname( plugin_basename( __FILE__ ) ) . '/languages/' );
 
 	global $wp_version;
 
-	$dependency_check = new Yoast_WooCommerce_Dependencies();
+	$dependency_check = new Classes\Dependencies();
 	if ( $dependency_check->check_dependencies( $wp_version ) ) {
 		global $yoast_woo_seo;
 
 		// Initializes the plugin.
-		$yoast_woo_seo = new Yoast_WooCommerce_SEO();
+		$yoast_woo_seo = new Classes\Main();
 	}
 }
 
 if ( ! wp_installing() ) {
-	add_action( 'plugins_loaded', 'initialize_yoast_woocommerce_seo', 20 );
+	add_action( 'plugins_loaded', 'Yoast\WP\Woocommerce\initialize', 20 );
```

Comment:
You _could_ use the below here (not a necessity though, choose whichever you prefer):
```suggestion
	add_action( 'plugins_loaded', __NAMESPACE__ . '\initialize', 20 );
```